<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyToolsCollection</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabRelease">
       <attribute name="title">
        <string>Release Actions</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pbUpdateMcgFwVersions">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Update MCG Firmware Versions</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>New MCG Fw Version:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbMcgFwVersion">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAtTop</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pbUpdateMcgPackVersions">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Update MCG Pack Versions</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="minimumSize">
               <size>
                <width>130</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>New Version:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbMcgPackVersions">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAtTop</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="pbCreateMcgPackTags">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Create MCG Pack Tags</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Branch Version:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbPackSrcBranchVersion">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAtTop</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Tag Version (Base):</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbMcgPackTagVersion">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAtTop</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="pbCopyMcgFwToInstLoc">
              <property name="minimumSize">
               <size>
                <width>210</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Copy MCG Firmware To Install Loc</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Source Version:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbMcgFwRepoVersion">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAtTop</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Target Version:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbMcgFwInstVersion">
              <property name="minimumSize">
               <size>
                <width>75</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="insertPolicy">
               <enum>QComboBox::InsertAtTop</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Logs:</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_6">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pbClearLog">
              <property name="text">
               <string>Clear Log</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTextEdit" name="teLog">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>895</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>pbUpdateMcgFwVersions</tabstop>
  <tabstop>pbUpdateMcgPackVersions</tabstop>
  <tabstop>pbCreateMcgPackTags</tabstop>
  <tabstop>pbCopyMcgFwToInstLoc</tabstop>
  <tabstop>pbClearLog</tabstop>
  <tabstop>teLog</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
